cmd/package_managers.go

```diff
@@ -129,23 +125,24 @@ type rubyGemsSearchResults struct {
 
 // Gets the GitHub repository URL for the npm package.
 func fetchGitRepositoryFromNPM(packageName string, packageManager pmc.Client) (string, error) {
-	npmSearchURL := "https://registry.npmjs.org/-/v1/search?text=%s&size=1"
+	npmSearchURL := "https://registry.npmjs.org/%s/latest"
```

Comment:
nit: we're no longer using the search API, so the URL shouldn't be `npmSearchURL`

Reply:
fixed!